constraints.txt

```diff
@@ -0,0 +1,83 @@
+# Constrains for testing Diazo / Plone Packages
```

Comment:
I do not like the approach of pinning versions here. As there is no automatic way to get them app to date.